Back out "[pytorch][PR] Revert "Move CreateContext to global registry (#11688)""

jerryzh168 opened the pull request:
Stack:
&nbsp;&nbsp;&nbsp;&nbsp;:black_circle:&nbsp; **#12121 Back out "[pytorch][PR] Revert "Move CreateContext to global registry (#11688)""**&nbsp;&nbsp;[:green_heart:](https://our.internmc.facebook.com/intern/diff/D10033396/)
&nbsp;&nbsp;&nbsp;&nbsp;:white_circle:&nbsp; #12029 [pt1][tensor] New in StaticContext returns at::DataPtr&nbsp;&nbsp;[:yellow_heart:](https://our.internmc.facebook.com/intern/diff/D9889990/)

Original commit changeset: 6ca9de65b707

Differential Revision: [D10033396](https://our.internmc.facebook.com/intern/diff/D10033396/)

ezyang reviewed (approved):
Approved via ghapprove D10033396

ezyang reviewed (approved):
Approved via `ghapprove D10033396`

ezyang reviewed (approved):
Approved via `ghapprove -s D10033396`

ezyang reviewed (approved):
Approved via `ghapprove -s D10033396`

jerryzh168 commented:
@pytorchbot retest this please

jerryzh168 commented:
@pytorchbot retest this please

jerryzh168 commented:
@pytorchbot retest this please

jerryzh168 committed:
Back out "[pytorch][PR] Revert "Move CreateContext to global registry (#11688)"" (#12121)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/12121

Pull Request resolved: https://github.com/pytorch/pytorch/pull/12055

Original commit changeset: 6ca9de65b707

Reviewed By: ezyang

Differential Revision: D10033396

fbshipit-source-id: 0936ccbeae529119586e41dd1a687f5ef1f85c8f
